packages/util-waiter/src/utils/validate.spec.ts

```diff
@@ -50,6 +50,17 @@ describe(validateWaiterOptions.name, () => {
     }
   });
 
+  it("should not throw an error with small decimal numbers", () => {
+    waiterOptions.maxWaitTime = 0.5;
+    waiterOptions.minDelay = 0.0001;
+    waiterOptions.maxDelay = 0.4;
+    try {
+      validateWaiterOptions(waiterOptions);
+    } catch (e) {
+      expect(e).toBe("SHOULD NOT ERROR HERE");
+    }
```

Comment:
This catch block is unnecessary. 

The try-branch, if not throwing, will not assert anything. The catch block assertion will never pass. 

That means having no try-catch block at all would produce the same result. If you just want to assert that some code does not throw, simply run it with no assertions.